src/components/FormSubmit/index.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import {View} from 'react-native';
+import stylePropTypes from '../../styles/stylePropTypes';
+
+const propTypes = {
+    /* A function to execute when form is submitted with ENTER */
+    onSubmit: PropTypes.func.isRequired,
+
+    /** Children to wrap with FormSubmit. */
+    children: PropTypes.node.isRequired,
+
+    /** Container styles */
+    style: stylePropTypes,
+};
+
+const defaultProps = {
+    style: [],
+};
+
+class FormSubmit extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.onKeyDownCapture = this.onKeyDownCapture.bind(this);
+    }
+
+    onKeyDownCapture(event) {
```

Comment:
The function should be named based on what they do. Please rename it.

Reply:
Could you suggest a name?

Reply:
What about `submitForm`?